tests/serial/mapping-nearest-projection/helpers.hpp

```diff
@@ -7,12 +7,32 @@
 using namespace precice;
 using precice::testing::TestContext;
 
-void testMappingNearestProjection(bool defineEdgesExplicitly, const std::string configFile, const TestContext &context);
+void testMappingNearestProjection(bool defineEdgesExplicitly, bool useBulkFunctions, const std::string configFile, const TestContext &context);
 
-void testQuadMappingNearestProjection(bool defineEdgesExplicitly, const std::string configFile, const TestContext &context);
+void testQuadMappingNearestProjection(bool defineEdgesExplicitly, bool useBulkFunctions, const std::string configFile, const TestContext &context);
 
-void testQuadMappingNearestProjectionTallKite(bool defineEdgesExplicitly, const std::string configFile, const TestContext &context);
+void testQuadMappingNearestProjectionTallKite(bool defineEdgesExplicitly, bool useBulkFunctions, const std::string configFile, const TestContext &context);
 
-void testQuadMappingNearestProjectionWideKite(bool defineEdgesExplicitly, const std::string configFile, const TestContext &context);
+void testQuadMappingNearestProjectionWideKite(bool defineEdgesExplicitly, bool useBulkFunctions, const std::string configFile, const TestContext &context);
+
+inline void testMappingNearestProjection(bool defineEdgesExplicitly, const std::string configFile, const TestContext &context)
+{
+  testMappingNearestProjection(defineEdgesExplicitly, false, configFile, context);
+}
+
+inline void testQuadMappingNearestProjection(bool defineEdgesExplicitly, const std::string configFile, const TestContext &context)
+{
+  testQuadMappingNearestProjection(defineEdgesExplicitly, false, configFile, context);
+}
+
+inline void testQuadMappingNearestProjectionTallKite(bool defineEdgesExplicitly, const std::string configFile, const TestContext &context)
+{
+  testQuadMappingNearestProjectionTallKite(defineEdgesExplicitly, false, configFile, context);
+}
+
+inline void testQuadMappingNearestProjectionWideKite(bool defineEdgesExplicitly, const std::string configFile, const TestContext &context)
+{
+  testQuadMappingNearestProjectionWideKite(defineEdgesExplicitly, false, configFile, context);
+}
```

Comment:
The alternative would be to port all tests do the 4-argument functions, right?
If yes, why did you choose this solution? (not saying it is bad, just to understand ... maybe we need some "why" documentation here)

Reply:
The inline functions were helpers during the migration. I removed them.

This is what you meant, right?

Reply:
yes :+1: